### ISSUE TITLE
Start time and execution time are not showed in the report HTML generated

### ISSUE THREAD

**Issue by ghost** [badeball/cypress-cucumber-preprocessor#798]
### Current behaviour

After adding the configuration below in the **.cypress-cucumber-preprocessorrc.json** file
```
{
"html": {
        "enabled": true,
        "output": "reports/cucumber-report.html"
    }
}
```

I see Uknowwn start time of the last run and Uknown duration 

### Desired behaviour

I should see the start time and duration of the execution

### Test code to reproduce

![image](https://user-images.githubusercontent.com/110224377/184235016-4f8db739-5d30-4c10-a169-fc4856356fb6.png)

### Versions

* **Cypress version**: 10.3.1
* **Preprocessor version**: 12.0.1
* **Node version**: 16.16.0

### Checklist

- [Yes ] I've read the [FAQ](https://github.com/badeball/cypress-cucumber-preprocessor/blob/master/docs/faq.md).
- [ yes] I've read [Instructions for logging issues](https://github.com/badeball/cypress-cucumber-preprocessor/blob/master/CONTRIBUTING.md#instructions-for-logging-issues).
- [ Yes] I'm not using `cypress-cucumber-preprocessor@4.3.1` (package name has changed and it is no longer the most recent version, see [#689](https://github.com/badeball/cypress-cucumber-preprocessor/issues/689)).

**Comment by badeball** [comment]
Hey, @diegoParser. Thanks for reporting this issue. It has been fixed in v12.1.0.

**Comment by ghost** [comment]
Thanks to you for your enormous effort in maintaining this project; it was a pleasure to apport at least a little reporting this bug.
